test/fast/Aliases/lts/"nvm alias" should ensure LTS alias dir exists

```diff
@@ -7,10 +7,23 @@ set -ex
 
 LTS_ALIAS_PATH="$(nvm_alias_path)/lts"
 
-die () { echo "$@" ; exit 1; }
+if [ -d "${LTS_ALIAS_PATH}" ]; then
```

Comment:
There's also `setup_dir` and `teardown_dir` if you want it just once for the entire dir (which is probably fine here)